js/app.js

```diff
@@ -28,74 +31,102 @@ alert(`Oh! ${userName}! I didn't recognize you. Come on in.`);
 // Start the game
 alert('Let\'s play a game. Answer some trivia about me. Go for a high score! Or a low score if you\'re too afraid of success. Acceptable answers are in the form yes/no and y/n, unless otherwise noted.');
 
-//Initialize a user answer
-let userAnswer = '';
+quizGame();
+
+//This function returns a random integer between min and max (both included)
+
+function randomInteger(min, max){
+  return Math.floor(Math.random() * (max - min + 1 )) + min;
+
+}
+
+function quizGame (){
```

Comment:
I REALLY like that you wrapped your entire game up into one function! Great work!